e2e/cypress/tests/integration/scroll/channel_scroll_spec.js

```diff
@@ -76,6 +75,37 @@ describe('Scroll', () => {
         });
     });
 
+    it('MM-T2367 Compact view', () => {
+        const message = 'This is the first post';
+
+        // # Set display mode to standard
+        cy.apiSaveMessageDisplayPreference('clean');
+
+        // # Post a starting message with user 1
+        Cypress._.times(2, () => {
+            cy.postMessage(message);
+        });
+
+        // * Verify that messages do no have user's name in post
+        cy.getLastPostId().then((parentMessageId) => {
+            cy.get(`#${parentMessageId}_message`).parent().invoke('text').then((text) => {
+                expect(text).to.equal(message);
+                expect(text).to.not.have.string('sysadmin');
+            });
+        });
+
+        // # Set display mode to compact
+        cy.apiSaveMessageDisplayPreference('compact');
+        cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
+
+        // * Verify that messages have user's name in post
+        cy.getLastPostId().then((parentMessageId) => {
+            cy.get(`#${parentMessageId}_message`).parent().invoke('text').then((text) => {
+                expect(text).to.equal(`sysadmin: ${message}`);
```

Comment:
The `Update your status` button seems to be causing issues on this assertion for me.

```suggestion
                expect(text).to.contain('sysadmin');
                expect(text).to.contain(message);
```

Reply:
Thank you @Phrynobatrachus .
I have made the suggested changes.